drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -649,9 +689,18 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             // Set up event handlers:
 
-            $element.on('click', '.item-feedback-popup .close-feedback-popup-button', closePopupEventHandler);
-            $element.on('keydown', '.item-feedback-popup .close-feedback-popup-button', closePopupKeydownHandler);
-            $element.on('keyup', '.item-feedback-popup .close-feedback-popup-button', preventFauxPopupCloseButtonClick);
+            // Depending on media setup the close button handlers.
+            if (window.matchMedia( MEDIA_QUERY ).matches) {
```

Comment:
Here I would just bind events to both buttons - the one that is hidden is not going to receive them anyway.